aardvark/persistence/sqlalchemy/__init__.py

```diff
@@ -0,0 +1,304 @@
+import datetime
+import logging
+from contextlib import contextmanager
+from typing import Any, Dict, List, Optional
+
+import confuse
+from sqlalchemy import create_engine, engine
+from sqlalchemy import func as sa_func
+from sqlalchemy.exc import SQLAlchemyError
+from sqlalchemy.orm import scoped_session, sessionmaker
+
+from aardvark.exceptions import CombineException, DatabaseException
+from aardvark.persistence import PersistencePlugin
+from aardvark.persistence.sqlalchemy.models import (AdvisorData, AWSIAMObject,
+                                                    Base)
+
+log = logging.getLogger("aardvark")
```

Comment:
nitpick, I'd have the log defined in a main file so you can just import it. (similar to some of our other tooling atm)

Reply:
I often have a hard time with that pattern because it tends to cause circular imports. This way is pretty low-cost (it only runs on import) and it returns a reference to the same logger instance.